src/coreclr/src/jit/hwintrinsic.h

```diff
@@ -100,9 +95,17 @@ enum HWIntrinsicFlag : unsigned int
     // No Read/Modify/Write Semantics
     // the intrinsic doesn't have read/modify/write semantics in two/three-operand form.
     HW_Flag_NoRMWSemantics = 0x4000,
+
+    // NoContainment
+    // the intrinsic cannot be handled by comtainment,
+    // all the intrinsic that have explicit memory load/store semantics should have this flag
+    HW_Flag_NoContainment = 0x40,
```

Comment:
This is misordered with respect to the rest of the flags.

Reply:
Yep, I meant to update this before marking this PR as ready for review.